d3a_api_client/redis.py

```diff
@@ -264,6 +261,13 @@ def executor_function():
             self.on_finish(message)
         self.executor.submit(executor_function)
 
+    def _check_buffer_message_matching_command_and_id(self, message):
+        transaction_id = message["transaction_id"]
```

Comment:
Can you also check and raise an appropriate exception if `transaction_id` is None or if `transaction_id` key not part of message ?

Reply:
yes! Good idea! Thanks!